platemate/food/views.py

```diff
@@ -272,6 +279,37 @@ class SubmissionForm(forms.Form):
     "admin": ["2011-04-10", "2011-04-11"],
 }
 
+@csrf_exempt
```

Comment:
I made this action csrf_exempt, assuming we'll rely on the api key. Is this reasonable?